### DISCUSSION TITLE
Mpv player as multimedia modul has no sound on Linux Mint 21.2

### DISCUSSION THREAD

**Discussion by stefan-franz** [smplayer-dev/smplayer#813] [Q&A]
I have more devices with the same Linux Mint and at one device there is no sound if i use as multimedia modul the mpv player. 
What can i do to fix this?
Installed versions: SMPlayer 23.6.0 - normal (not flathub) installation and the mpv player 0.34.1-1ubuntu3

**Comment by smplayer-dev** [comment]
Try selecting another audio device in preferences -> general -> audio.

**Comment by stefan-franz** [comment]
Was my mistake. Changing the player caused the volume control to nearly zero. I've only seen it now. Everything works fine now.
